### ISSUE TITLE
Multiple subscription

### ISSUE THREAD

**Issue by midesweb** [LimeDeck/nova-cashier-overview#6]
I am trying to manage different subscriptions on the same resource:

```php
public function fields(Request $request)
{
    return [
        ID::make()->sortable(),
        // ...

        Subscription::make('one'),
        Subscription::make('two'),

        // ...
    ];
}
```

However, when the related resource is opened in Nova the system only shows the first subscription field (In this case "one").

**Comment by rudowastaken** [comment]
Hi @midesweb, I suspect that Nova displays only a single Resource Tool with the same name. I'd appreciate if somebody knows whether it's really enforced like this or there is a way around it. 

Changing the name of the subscription was originally meant to allow for different naming conventions. I haven't really needed to use this since we usually link multiple billable entities for User (Project,...) when creating subscriptions but this feature could also be interesting to some people.

**Comment by eduardocruz** [comment]
Got the same issue. I have more than one subscription name, and it does not work, since the package allows only a single subscription name.

**Comment by mstaack** [comment]
yeah same for me, i have multiple subscription but only one per user, displaying the correct one does not work.
